inst/rmarkdown/templates/lncs/skeleton/llncs.cls

```diff
@@ -0,0 +1,1218 @@
+% LLNCS DOCUMENT CLASS -- version 2.21 (12-Jan-2022)
```

Comment:
is the CTAN one not updated by the publisher ? Do you think is not following new version ? 

The package seems new but maintained by markus richter, which seems to be from Springer https://www.springerprofessional.de/en/markus-richter/12484082

Usually when a CTAN package is available it is better to use it so that it is easier to get updated version in the future.

Maybe we should confirm with the publisher ?